content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst

```diff
@@ -23,7 +23,7 @@ three step deliveries. First, make sure the *Multi-Step Routes* option is enable
 :menuselection:`Inventory --> Configuration --> Settings --> Warehouse`. Note that activating
 :guilabel:`Multi-Step Routes` will also activate *Storage Locations*.
 
-.. image:: delivery_three_steps/multi-step-routes.png
+.. image:: delivery_three_steps/delivery-three-steps-multi-step-routes.png
```

Comment:
I don't see an improvement here with the new image. Can you help me understand why there was a change?